ApptentiveFeedback/ApptentiveFeedback/Additions/ApptentiveCallbacks.cs

```diff
@@ -0,0 +1,234 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Android.App;
+using Android.Content;
+using Android.Runtime;
+using Android.Widget;
+using ApptentiveSDK.Android;
+using Java.Interop;
+using Apptentive.Com.Android.Util;
+using Apptentive.Com.Android;
+using ApptentiveCore;
+using Java.Security;
+using Apptentive.Com.Android.Feedback.Engagement.Interactions;
+using Android.OS;
+
+
+namespace ApptentiveSDK.Android
+{
+    public partial class Apptentive
+    {
+    
+        /// <summary>
+        /// Registers the ApptentiveKit.Android SDK.
+        /// </summary>
+        /// <param name="application"> The main application.</param>
+        /// <param name="configuration">The ApptentiveConfiguration object.</param>
+        /// <param name="onCompletion">Called after successful or failed registration is conducted.</param>
+
+        public static void Register(Application application, ApptentiveConfiguration configuration, Action<bool> onCompletion = null)
+        {
+            configuration.DistributionName = "Xamarin";
+            configuration.DistributionVersion = "6.0.4";
```

Comment:
Frank had a recommendation instead of hardcoding the value here. Did you try that? It should be in the conversation thread.